zulip_bots/zulip_bots/bots/file_uploader/doc.md

```diff
@@ -0,0 +1,3 @@
+# File Uploader Bot
+
+This bot allows user to upload file on given path to Zulip server.
```

Comment:
This is a simple bot, but this seems a little short; perhaps include usage instructions here, as per other `doc.md` files, and maybe discuss limitations of uploading, if there are any?

Reply:
Added an example of using the File Uploader Bot in `doc.md`

Reply:
That's an improvement; the `doc.md` files for other bots show the typical structure in use - eg. subheadings and other information they include. As with coding, if you can follow the current (majority) style that is good practice :)

Reply:
I updated `doc.md` (used `dropbox` bot's documentation as an example)